File: find_team_data.js
const fs = require('fs');
const path = require('path');
const glob = require('glob');

// Since glob might not be installed in the root node_modules, we can try to use a simple recursive directory search
// or assume we are in an environment where we can just read the specific directory if we know it.
// simpler: just read the known directory public/sitnovate-app/assets

const searchDir = path.join(__dirname, 'public', 'sitnovate-app', 'assets');

try {
    const files = fs.readdirSync(searchDir).filter(f => f.endsWith('.js'));
    const targets = ["Abhani", "Raipurkar", "Team"];

    console.log(`Searching in ${searchDir}...`);

    files.forEach(file => {
        const filePath = path.join(searchDir, file);
        try {
            const content = fs.readFileSync(filePath, 'utf8');
            let found = false;

            targets.forEach(t => {
                if (content.includes(t)) {
                    if (!found) {
                        console.log(`\n--- Found matches in ${file} ---`);
                        found = true;
                    }
                    console.log(`MATCH: '${t}'`);
                    const idx = content.indexOf(t);
                    const start = Math.max(0, idx - 300);
                    const end = Math.min(content.length, idx + 800);
                    console.log(`CONTEXT wrapped: ...${content.slice(start, end)}...`);
                }
            });
        } catch (err) {
            console.error(`Error reading ${file}: ${err.message}`);
        }
    });
} catch (err) {
    console.error(`Error reading directory: ${err.message}`);
}

File: find_team_data.py
import glob
import os

# Use absolute path for search to be safe, or relative to cwd
search_path = r"public/sitnovate-app/assets/*.js"
files = glob.glob(search_path)

targets = ["Abhani", "Raipurkar", "Team"]

print(f"Searching in {search_path}...")
for f in files:
    try:
        with open(f, 'r', encoding='utf-8') as file:
            content = file.read()
            found = False
            for t in targets:
                if t in content:
                    if not found:
                         print(f"\n--- Found matches in {os.path.basename(f)} ---")
                         found = True
                    print(f"MATCH: '{t}'")
                    idx = content.find(t)
                    start = max(0, idx - 300)
                    end = min(len(content), idx + 800)
                    print(f"CONTEXT wrapped: ...{content[start:end]}...")
    except Exception as e:
        print(f"Error reading {f}: {e}")
